src/exceptionite/renderers/JavascriptRenderer.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
what will this javascript renderer do?

Reply:
There are 2 renderers for now: Terminal and Javascript (maybe it could be called HTMLRenderer or WebRenderer as this is mainly used in web frameworks).
This is the renderer used in Masonite or Flask to display the exception page.

Reply:
See here the usage for Masonite:
https://github.com/MasoniteFramework/masonite/pull/522/files#r800042033